src/metabase/search/config.clj

```diff
@@ -123,6 +129,19 @@
   {:arglists '([model])}
   (fn [model] model))
 
+(defmethod columns-for-model "action"
+  [_]
+  (conj default-columns :model_id
+        [{:select [:name]
+          :from   [:report_card]
+          :where  [:= :id :action.model_id]}
+         :model_name]
+         ;; Can't select more than one column in a subselect, thus the duplication
+        [{:select [:collection_id]
+          :from   [:report_card]
+          :where  [:= :id :action.model_id]}
+         :collection_id]))
```

Comment:
Why do you need a subselect though? You're left joining on report_card, so you can select columns from `:model` directly

`(sql.helpers/left-join [:report_card :model]
                             [:= :model.id :action.model_id])`

Reply:
total brainfart on my part, really good catch, thanks!